python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -418,6 +418,15 @@ def materialized() -> "AssetCondition":
 
         return MaterializedSchedulingCondition()
 
+    @staticmethod
+    def in_latest_time_window() -> "AssetCondition":
+        """Returns an AssetCondition that is true for an asset partition when it is within the latest
+        time window.
+        """
+        from .slice_condition import InLatestTimeWindowCondition
+
+        return InLatestTimeWindowCondition()
+
```

Comment:
worth explaining the behavior of this in alll of the partitioning schema. Maybe ideal thing is to write a doc and then reference it here. Can do that in followup